prime-router/src/main/kotlin/history/db/ReportGraph.kt

```diff
@@ -0,0 +1,165 @@
+package gov.cdc.prime.router.history.db
+
+import gov.cdc.prime.router.azure.DatabaseAccess
+import gov.cdc.prime.router.azure.db.Tables.COVID_RESULT_METADATA
+import gov.cdc.prime.router.azure.db.Tables.REPORT_FILE
+import gov.cdc.prime.router.azure.db.Tables.REPORT_LINEAGE
+import gov.cdc.prime.router.azure.db.tables.pojos.CovidResultMetadata
+import gov.cdc.prime.router.common.BaseEngine
+import org.apache.logging.log4j.kotlin.Logging
+import org.jooq.CommonTableExpression
+import org.jooq.Record
+import org.jooq.Record2
+import org.jooq.impl.DSL
+import org.jooq.impl.DSL.selectDistinct
+import org.jooq.impl.SQLDataType
+import java.util.UUID
+
+private const val PARENT_REPORT_ID_FIELD = "parent_report_id"
```

Comment:
Kept these here, but private. Let me know if you prefer me to also move them into the class